content/project/single-module-application/src/main/java/${project_rootPath}/${project_classPrefix}Application.java

```diff
@@ -22,4 +32,12 @@ public static void main(String[] args) {
 	DataSource dataSource() {
 		return DataSourceBuilder.create().build();
 	}
+
+	@EventListener
+	public void printApplicationUrl(ApplicationStartedEvent event) {
+		LoggerFactory.getLogger(${project_classPrefix}Application.class).info("Application started at "
+				+ "http://" + InetAddress.getLoopbackAddress().getHostName() + ":"
```

Comment:
What about to simplify to `http://localhost:`?
I think for our purpose it will always work.

Reply:
Agreed, this should make it easier to understand what is going on in this method.